spec/features/ingredients/edit_ingredient_page_spec.rb

```diff
@@ -0,0 +1,35 @@
+feature 'user visits edit ingredients page' do
+  before do
+    FactoryBot.create(:ingredient, :egg)
+    visit('/ingredients')
+    click_link('Edit')
+    current_path.match(%r{/ingredients/\d+/edit})
```

Comment:
You can do direct match:

```ruby
ingredient = FactoryBot.create(:ingredient, :egg)
current_path.match(%r{/ingredients/#{ingredient.id}/edit})
```

We don't even need this matching here because:

1. You don't have assertion, therefore even if this line evaluates to `false` test will still pass.
2. `before` block is not a place for assertions.